app/src/main/java/com/github/fribourgsdp/radio/GameActivity.kt

```diff
@@ -19,11 +25,16 @@ class GameActivity : AppCompatActivity(), GameView {
     private lateinit var songTextView : TextView
     private lateinit var errorOrFailureTextView : TextView
     private lateinit var songGuessEditText : EditText
+    private lateinit var muteButton : ImageButton
     private lateinit var songGuessSubmitButton: Button
 
     private lateinit var playersListView : ListView
     private lateinit var namesAdapter : ArrayAdapter<String>
 
+    private lateinit var playerGameHandler: PlayerGameHandler
```

Comment:
The `playerGameHandler` is only used in `onCreate` so no need to extract it.

Reply:
No it is also called in initViews() :)

Reply:
Indeed, this has been corrected.